docs/topics/db/optimization.txt

```diff
@@ -261,44 +261,51 @@ But:
 
 .. _overuse_of_count_and_exists:
 
-Don't overuse ``count()`` and ``exists()``
-------------------------------------------
+Don't overuse ``contains()``, ``count()``, and ``exists()``
+-----------------------------------------------------------
 
 If you are going to need other data from the QuerySet, evaluate it immediately.
```

Comment:
I think this sentence could be clearer, maybe:

If you will eventually need to evaluate a queryset, evaluate it upfront so that you can utilise the cached result.

Reply:
I like the current version somehow :thinking: , please send a separate PR if you think otherwise.